Makefile

```diff
@@ -302,3 +302,7 @@ bump-version:
 	sed -i "s|Release:\(\s*\)[[:digit:]]\+|Release:\11|" osbuild.spec
 	sed -i "s|version=\"$(VERSION)\"|version=\"$(NEXT_VERSION)\"|" setup.py
 	sed -i "s|__version__ = \"$(VERSION)\"|__version__ = \"$(NEXT_VERSION)\"|" osbuild/__init__.py
+
+.PHONY: format
+format:
+	black osbuild/ assemblers/* devices/* inputs/* mounts/* runners/* sources/* stages/*
```

Comment:
What about `tools`? Shouldn't we just `black .`?

Reply:
`tools/` should be added. Doing `black .` will only format `*.py` files recursively; our modules generally don't end in `.py` (see also `$LINTABLES` in `tox.ini`).

Reply:
Ahhh, thanks for the explanation. I wonder if `black .` instead of `black osbuild/` would be slightly more future-proof. :thinking: